qabelbox/src/main/java/de/qabel/qabelbox/activities/MainActivity.java

```diff
@@ -249,9 +253,66 @@ protected void onCreate(Bundle savedInstanceState) {
 
         bindService(serviceIntent, mServiceConnection, Context.BIND_AUTO_CREATE);
 
+        setConnectivityManager(new ConnectivityManager(this));
         addBackStackListener();
     }
 
+    public void setConnectivityManager(ConnectivityManager manager) {
```

Comment:
Bad method name. It doesn't just set it, it sets up listeners and does stuff. What about 'installConnectivityManager'?